### PR TITLE
Error Responses content changed

### PR DESCRIPTION
**Pull request by RubenBG7**
According to Issue Aggreement

### CODE REVIEW

**Comment by hdamker** [comment]
If the purpose of the document is no longer that of an example (indicated by the new title and description "Common data and errors for CAMARA APIs") it should contain only componentes/schemas which are already agreed as common across the sub projects. Which is only the error info schema, but not "MoneyAmount" and others.

**Comment by shilpa-padgaonkar** [comment]
AFAIK, it was meant to be an example. @RubenBG7 :Was this scope somehow changed ? If not, could you kindly rename the doc accordingly (for e.g. Camara_common_examples.json)?

File: Commonalities/documentation/CAMARA_common.json [hunk 1]
```diff
@@ -1,612 +1,439 @@
...
+      "description": "conflict",
+      "content": {
+        "application/json": {
+          "schema": {
+            "$ref": "#/components/schemas/ErrorInfo"
+          },
+          "example": {
+            "status": 409,
+            "code": "CONFLICT",
+            "message": "Schema validation failed at  ..."
```
Comment (PedroDiez) [inline Commonalities/documentation/CAMARA_common.json:156]:
```suggestion
            "message": "A specified resource duplicate entry found"

```

**Review by PedroDiez — COMMENTED** [review]
Some suggestions regarding HTTP "status", and "message" text

File: Commonalities/documentation/CAMARA_common.json [hunk 2]
```diff
@@ -1,612 +1,439 @@
 {
   "info": {
-    "description": "Common data and errors for Telefonica APIs",
-    "version": "0.1.0",
-    "title": "Common data and errors for Telefonica APIs",
-    "contact": {
-      "name": "API DIGITAL INTEGRATION",
-      "email": "apidigital.integration@telefonica.com"
-    }
+    "description": "Common data and errors for CAMARA APIs",
```
Comment (shilpa-padgaonkar) [inline Commonalities/documentation/CAMARA_common.json:3]: @RubenBG7 : Could you please change the description and title such that it clearly states that this is just a sample?

**Comment by shilpa-padgaonkar** [comment]
@RubenBG7 : Will your new PR https://github.com/camaraproject/WorkingGroups/pull/176 make this one obsolete?
If yes, kindly confirm so that this PR can be closed.

**Comment by RubenBG7** [comment]
Confimed @shilpa-padgaonkar  PR #176 make these ones obseolete

**Pull request closed without merging**